sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class JaegerRemoteSamplerProvider implements ConfigurableSamplerProvider {
+
+  private static final String ENDPOINT_KEY = "endpoint";
+  private static final String POLLING_INTERVAL = "pollingInterval";
+  private static final String INITIAL_SAMPLING_RATE = "initialSamplingRate";
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    String serviceName = config.getString("otel.service.name");
+    JaegerRemoteSamplerBuilder builder = JaegerRemoteSampler.builder().setServiceName(serviceName);
+    Map<String, String> params = parseArg(config.getString("otel.traces.sampler.arg"));
```

Comment:
Just noticed - can this be `config.getCommaSeparatedMap`?

Like https://github.com/open-telemetry/opentelemetry-java-contrib/pull/67/files